pages/permission_management.rst

```diff
@@ -0,0 +1,319 @@
+
+2) Creating and updating Roles (only Administrator users could do this)
+Initially, two roles existed: Administrator and Reader. Administrators could create custom roles. Additionally Administrators needed to assign Roles to Profiles individually because the only access new users obtained was to the Graylog platform with no entities assigned to them. 
+
+The roles could then be assigned to one or more users, either manually or as part of the synchronization step from Active Directory/LDAP. However, this integration only provided users access to Graylog using the organization’s authoritative identity source, but it lacked the ability to give permissions within Graylog. Administrators needed to add those permissions manually. 
+
+Synchronizing with AD/LDAP was called “Group Mapping” and required configuration of the AD/LDAP search query for groups in the identity provider. Group mapping then contained the information on which external group to map to which role. On each login for a user, the system would reset the roles assigned to this user based on the group membership in the authoritative identity source.
+
+Roles only had UI for granting access to Streams and Dashboards, the backend API could be used to specify other permissions as well, but this required knowledge about the permission system’s implementation.
+
+Since only Administrators could edit roles, this process gave Administrators complete access to everything which impeded the ability to limit privileged access according to the principle of least privilege. Additionally, this process also created a proliferation of roles depending on how organizations structured them. 
+
+Graylog 4.0 removes this functionality and problem. 
+
+.. image:: /images/permissions/permissions_03.png
+
+3) “View sharing” that only applied to saved searches and dashboards (users with existing access to that entity could re-share it)
+“View” is the internal name for the underlying system for Search and Dashboards. “Stream search” is a term that appears in several places, but that is simply a search that is restricted to a certain dataset. Streams are the fundamental unit for data access control in Graylog.
+
+.. image:: /images/permissions/permissions_04.png
+
+Except for the “view sharing” dialog, administrators had to grant permissions from a central location (the roles). This meant that administrators had extensive access within Graylog making it difficult to limit access according to the principle of least privilege. 
+
+When a user created a new dashboard and reports, the entities would be visible to all users. For smaller organizations with few users, this becomes a hassle but remains manageable. For larger organizations, this functionality often created an overabundance of reports which made finding the needed ones cumbersome. 
+
+All of these features were available in Graylog Open Source, none of the authentication features were ever an Enterprise offering.
```

Comment:
I would not include this section at this point (maybe not at all), because the documentation is about 4.0 and not 3.3.
This might be worthwhile to put into a blog post to highlight how things changed.
Most of this was written as background to inform tech writing, not for inclusion in the documentation.

Reply:
we could move that in a sub page and refer that as changes compared to 3.3 and so it is easier to remove that later.

Reply:
I will drop the 3.3 paragraph.